docs/build/configure/triggers.md

```diff
@@ -252,10 +295,13 @@ The request includes the following headers:
 <!-- prettier-ignore -->
 | Header Key | Description |
 | ---------- | ----------- |
-| `Part-ID` |  The part of the machine that triggered the request. |
-| `Robot-ID` | The machine that triggered the request. |
-| `Location-ID` | The location of the machine that triggered the request. |
-| `Org-ID` | The organization that triggered the request. |
+| `Org-Id` | The ID of the organization that triggered the request. |
```

Comment:
@maxhorowitz : I've not been able to confirm these all work:

Here is an example of a trigger from ingested binary data. I don't get the associated component type, name, method name, and min-time received, max time received.
```
127.0.0.1 - - [18/Jul/2024 17:49:42] "POST /?data_type=binary&event_type=part_data_ingested HTTP/1.1" 200 -
ImmutableMultiDict([('data_type', 'binary'), ('event_type', 'part_data_ingested')])
Host: d99b-71-167-222-76.ngrok-free.app
User-Agent: Go-http-client/2.0
Content-Length: 375
Accept-Encoding: gzip
Content-Type: application/json
Location-Id: vw3iu72d8n
Org-Id: bccf8f8f-e3c4-4f72-ab9a-fc547757f352
Part-Id: 422afb68-21d0-487c-a135-01d81ff54407
Robot-Id: 584b3120-0da5-4014-b843-8a685e9cab2d
X-Forwarded-For: 2600:1900:2000:a3::1:900
X-Forwarded-Host: d99b-71-167-222-76.ngrok-free.app
X-Forwarded-Proto: https
```

Reply:
And then here's an example of `part_online`:

```
127.0.0.1 - - [18/Jul/2024 17:48:12] "GET /?event_type=part_online HTTP/1.1" 200 -
ImmutableMultiDict([('data_type', 'binary'), ('event_type', 'part_data_ingested')])
Host: d99b-71-167-222-76.ngrok-free.app
User-Agent: Go-http-client/2.0
Content-Length: 375
Accept-Encoding: gzip
Content-Type: application/json
Location-Id: vw3iu72d8n
Org-Id: bccf8f8f-e3c4-4f72-ab9a-fc547757f352
Part-Id: 422afb68-21d0-487c-a135-01d81ff54407
Robot-Id: 584b3120-0da5-4014-b843-8a685e9cab2d
X-Forwarded-For: 107.178.207.105
X-Forwarded-Host: d99b-71-167-222-76.ngrok-free.app
X-Forwarded-Proto: https
```

Reply:
## 1st message -
component type, name, method name, min-time received, and max time received were moved to the data ingested POST request body - I am sorry if I grazed over this in a previous review - it was a change that was made during the introduction of POST request for part data ingested webhooks
![Screenshot 2024-07-22 at 4 35 59 PM](https://github.com/user-attachments/assets/54cf1876-ac24-4d3e-8f71-76978f1b7dd1)

## 2nd message -
https://viam.atlassian.net/browse/APP-5608